packages/librhsp/package.json

```diff
@@ -24,13 +26,12 @@
     },
     "scripts": {
         "build": "tsc",
-        "install": "node-gyp-build \"node scripts/build-RHSPlib.mjs\"",
+        "install": "pkg-prebuilds-verify binding-options.cjs || cmake-js compile",
```

Comment:
Note to self: Review this

Reply:
Looking deeper, this command specifically checks for a prebuild folder, not just a build folder. When we load it, it will accept a build folder.